packages/common/src/gckms/GckmsConfig.js

```diff
@@ -79,32 +65,48 @@ function getDefaultStaticConfig() {
   return defaultConfig;
 }
 
-function getNetworkName() {
-  if (isPublicNetwork(argv.network || "")) {
-    // Take everything before the underscore:
-    // mainnet_gckms -> mainnet.
-    return argv.network.split("_")[0];
+function getGckmsConfig(keys = arrayify(argv.keys), network = argv.network) {
+  let configOverride = {};
+
+  // If there is no env variable providing the config, attempt to pull it from a file.
+  // TODO: this is kinda hacky. We should refactor this to only take in the config using one method.
```

Comment:
> We should refactor this to only take in the config using one method.

You mean either via env variable or .GckmsOverride.js? If so, then I agree. Using env variable would be more consistent with the rest of our repo, but the env variable is very hard to parse and manipulate unlike the `.js` file.

Maybe @chrismaree or @daywiss has a better method of editing `.env` variables?

Reply:
Sorry, this code was just moved, not changed :). I didn't add this comment. I tried to keep this implementation as close to the original as possible to maintain backwards compatibility.

Reply:
Specifically on this issue, we use both in different places (the file locally and the env variable on GCP). We can definitely deprecate one, but I think that's separate from this PR.

The file works nicely locally because you can download it and forget it.

We have no good way of writing a file to cloud run, so env variables are more convenient. We could, however, host this file somewhere and have GCP download it, but that might be a little dangerous security-wise. Definitely more to think about there.

Reply:
ye IMHO having both is really useful and as @mrice32 pointed out envs work _much_ better in the context of cloud run.